corpus/reward/penalties--penalize-non-account-sender-multiple-miners.json

```diff
@@ -32,7 +32,7 @@
 	"apply_tipsets": [
 		{
 			"epoch": 1,
-			"basefee": "100",
+			"basefee": 100,
```

Comment:
Had to update these vectors because the schema changed slightly, and the previous vectors became unparseable.